src/test/java/core/basesyntax/FileWorkTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileWorkTest {
+    private static final String FIRST_FILE_NAME = "test1";
+    private static final String SECOND_FILE_NAME = "test2";
+    private static final String THIRD_FILE_NAME = "test3";
+
+    private static final String[] EMPTY_ARRAY_RESULT = new String[0];
+    private static final String[] ARRAY_RESULT = new String[]{"walk", "warm", "with", "with"};
```

Comment:
@Sofasmile please add test case for case sensitive values: wall Wave width World WWW